src/javascript/string-locator-instawp.js

```diff
@@ -0,0 +1,115 @@
+( function ( wp, $ ) {
+	'use strict';
+
+	if ( ! wp ) {
+		return;
+	}
+
+	$( function () {
+		$( document ).on( 'click', '.instawp-activate-now', function ( event ) {
+			const $button = $( event.target );
+			let slug = $button.data( 'slug' );
+
+			$button.text('Activating plugin ...');
+
+			event.preventDefault();
+
+			let data = {
+				action: 'install_activate_plugin',
+				plugin: slug
+			}; 
+
+			$.post(ajaxurl, data, function(response) {
+				if (response.success) {
+					$button.text(response.data.anchor_text);
+					$button.attr('href', response.data.href);
+					$button.attr('data-originaltext', response.data.anchor_text);
+					$button.attr('aria-label', response.data.anchor_text);
+					$button.attr('target', '_blank');
+					$button.removeClass('instawp-activate-now');
+					$button.addClass('string-locator-instawp-button disabled');
+				} else {
+					console.log('Error installing and activating the plugin:', response.data);
```

Comment:
The error handling here requires a developer console to be open, which the target audience is unlikely to have.

Reply:
Resolved in latest commit.